examples/proxy_attestation/module/rust/build.rs

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2020 The Project Oak Authors
+// Copyright 2021 The Project Oak Authors
```

Comment:
Random drive-by: leave copyright date as [date of creation](https://opensource.google/docs/copyright/#the-year), not date of last modification.

Reply:
It's a newly created file

Reply:
Ah, my bad, I made assumptions because it was showing as a diff.